### PR TITLE
Add a real blank page

### PR DESCRIPTION
**Pull request by JasonEtco**
💥 This page shows when users go to a channel without any activity. 

<details>
<summary>v1, see comments below for newer iterations</summary>

image

</details>

---

You'll notice there's no syntax highlighting on the Node.js client bit; I can pull in a React component for it, anyone have a favorite (of the thousands out there)?

### CODE REVIEW

**Comment by tcbyrd** [comment]
Awesome! I have no preferences, but it does seem like we're using highlight.js in [other places](https://github.com/probot/probot.github.io/blob/f255de17499ce5972f6dfc34abce6c354fa0ee49/index.html#L83-L92). If there's a way to pull in a subset of it like in [this example](https://github.com/probot/probot.github.io/blob/master/assets/css/syntax.css) that works as a React component, that would be my vote.

File: src/components/Blank.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { InfoIcon } from 'react-octicons'
+
+const code = `const SmeeClient = require('smee-client')
+
+const smee = new SmeeClient({
+  source: 'https://smee.io/abc123',
```
Comment (bkeepers) [inline src/components/Blank.js]: Could this be pre-populated with current channel url?

File: src/components/Blank.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            className="form-control input-xl input-block"
+          />
+          <p className="mt-2 text-gray-light f6">This page will automatically update as things happen.</p>
+
+          <hr />
+          <div className="mt-4 markdown-body">
+            <h3>Use the CLI</h3>
+            <p>The <code>smee</code> command will forward webhooks from smee.io to your local development environment.</p>
+            <pre><code>
+              $ smee
```
Comment (bkeepers) [inline src/components/Blank.js]: Same here, it'd be nice to show `smee -s https://smee.io/<channel-id>`

**Comment by bkeepers** [comment]
Also, what would you think about including directions for probot on this page? 😮

File: src/components/Blank.js (hunk 1 above)
Comment (JasonEtco) [reply src/components/Blank.js]: yesss great idea

**Comment by JasonEtco** [comment]
> Also, what would you think about including directions for probot on this page? 😮

What's Probot? `:trollface:` - yes, that's a good idea. Not sure about the wording though.

> If you're using Probot:
> ```
> npm install --save smee-client
> ```
> Then set the environment variable:
> ```
> WEBHOOK_PROXY_URL=https://smee.io/CHANNEL
> ```

**Comment by JasonEtco** [comment]
💥 Feedback implemented. 💥 

Note that the URLs it shows are all `localhost`, it uses `window.location.href` so its domain-agnostic. Looks a little odd in the Node.js example since the two URLs are the same, but ignore that 👍 

image

**Comment by bkeepers** [comment]
Looks great 👍

**Pull request merged**